README.md

```diff
@@ -1,6 +1,6 @@
 # pre-commit-hooks-ros
 
-ros-related hooks for pre-commit <https://pre-commit.com/>
+[ROS][]-related hooks for [pre-commit]
```

Comment:
Ah, mistake. :pleading_face: 

```suggestion
[ROS][]-related hooks for [pre-commit][]
```

Reply:
But without `[]` works as well? :thinking:

Reply:
Will check the official reference.

Reply:
I'll remove `[]`.

https://spec.commonmark.org/0.30/#example-556
https://github.github.com/gfm/#example-565